stdlib/number.gr

```diff
@@ -493,3 +951,102 @@ export let cos = (radians: Number) => {
     sin(pi / 2 + radians)
   }
 }
+
+/**
+ * Computes the tangent of a number (in radians) using Chebyshev polynomials.
+ *
+ * @param radians: The input in radians
+ * @returns The computed tangent
+ *
+ * @since v0.5.4
+ */
+export let tan = (radians: Number) => {
+  if (isNaN(radians) || isInfinite(radians)) {
+    nan
+  } else {
+    sin(radians) / cos(radians)
+  }
+}
+
+/**
+ * Computes the gamma function of a value using
+ *
+ * @param n: The value to compute the gamma of
+ * @returns The gamma of the given value
+ *
+ * @since v0.5.4
+ */
+// Math.gamma implemented using the stirling's approximation
+// https://en.wikipedia.org/wiki/Lanczos_approximation
```

Comment:
Putting a 2-line comment here broke the graindoc. Didn't you notice there was basically no output for this function?